test/e2e/network/netpol/network_policy.go

```diff
@@ -957,6 +947,83 @@ var _ = SIGDescribeCopy("Netpol [LinuxOnly]", func() {
 	})
 })
 
+var _ = SIGDescribeCopy("Netpol [Feature:SCTPConnectivity][LinuxOnly][Disruptive]", func() {
+	f := framework.NewDefaultFramework("sctp-network-policy")
+
+	ginkgo.BeforeEach(func() {
+		// Windows does not support network policies.
+		e2eskipper.SkipIfNodeOSDistroIs("windows")
+	})
+
+	ginkgo.Context("NetworkPolicy between server and client using SCTP", func() {
+		ginkgo.BeforeEach(func() {
+			addSCTPContainers = true
+			initializeResourcesByFixedNS(f)
+		})
+
+		ginkgo.AfterEach(func() {
+			addSCTPContainers = false
```

Comment:
wait if we set this to `true` before each test, is this reset to `false`... just a no-op ?

Reply:
I'm thinking in the scenario it runs it randomly, so I'm restoring to the last state after finishing the test.